ui/src/App.js

```diff
@@ -0,0 +1,135 @@
+import { Calendar, momentLocalizer } from "react-big-calendar";
+import moment from "moment";
+import "react-big-calendar/lib/css/react-big-calendar.css";
+import { useState } from "react";
+import s from "./App.module.css";
+
+const localizer = momentLocalizer(moment);
+const hours = new Date().getHours();
+const minutes = new Date().getMinutes();
+const padNumberWith0 = (num) => num.toString().padStart(2, "0");
+const defaultStartTime = `${padNumberWith0(hours)}:${padNumberWith0(minutes)}`;
+
+Date.prototype.addHours = function (h) {
+  this.setTime(this.getTime() + h * 60 * 60 * 1000);
+  return this;
+};
+
+const defaultEndTime = `${padNumberWith0(hours + 1)}:${padNumberWith0(
+  minutes
+)}`;
+
+function padTo2Digits(num) {
+  return num.toString().padStart(2, "0");
+}
+
+function formatDate(date) {
+  return [
+    date.getFullYear(),
+    padTo2Digits(date.getMonth() + 1),
+    padTo2Digits(date.getDate()),
+  ].join("-");
+}
+
+const formatAMPM = (timeString) => {
+  const date = timeString.substring(0, 5);
+  let hours = date.substring(0, 2);
+  let minutes = date.substring(3, 5);
```

Comment:
can we use Date.getDate() Date.getHour() and Date.getMinutes() here instead?

Reply:
This is working to our advantage at the moment. Decided on moving forward without digging to much into this.